compiler/rustc_feature/src/removed.rs

```diff
@@ -136,6 +136,9 @@ declare_features! (
      Some("subsumed by `#![feature(allocator_internals)]`")),
     /// Allows use of unary negate on unsigned integers, e.g., -e for e: u8
     (removed, negate_unsigned, "1.0.0", Some(29645), None, None),
+    /// Allows `#[no_coverage]` on functions.
+    /// The feature was renamed to `coverage` and the attribute to `#[coverage(on|off)]`
+    (removed, no_coverage, "CURRENT_RUSTC_VERSION", Some(84605), None, Some("renamed to `coverage`")),
```

Comment:
I only just noticed that this wasn't updated when the feature flag was renamed from `coverage` to `coverage_attribute`.

(This also affects the relevant UI tests.)

Reply:
I've filed PR #115832.